bundles/org.openhab.binding.homeconnect/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,676 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="homeconnect"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Home Connect API Bridge -->
+	<bridge-type id="api_bridge">
+		<label>Home Connect API</label>
+		<description>This bridge represents the gateway to the Home Connect API.</description>
+		<config-description>
+			<parameter name="clientId" type="text" required="true">
+				<label>Client id</label>
+				<description>Application client id</description>
+			</parameter>
+			<parameter name="clientSecret" type="text" required="true">
+				<label>Client secret</label>
+				<description>Application client secret</description>
+				<context>password</context>
+			</parameter>
+			<parameter name="simulator" type="boolean" required="true">
+				<label>Use simulator environment</label>
+				<description>Use simulated environment at https://developer.home-connect.com/simulator/</description>
+				<default>false</default>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<!-- Dishwasher -->
+	<thing-type id="dishwasher">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="api_bridge"/>
+		</supported-bridge-type-refs>
+		<label>Dishwasher</label>
+		<description>Home Connect connected dishwasher (e.g. Bosch or Siemens).</description>
+		<channels>
+			<channel id="power_state" typeId="power_state"/>
```

Comment:
There are predefined system channel types. https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types This could be `system.power`.

Reply:
Thanks, will change it.